lib/lita/adapters/slack.rb

```diff
@@ -9,7 +9,9 @@ class Slack < Adapter
       # Required configuration attributes.
       config :token, type: String, required: true
       config :proxy, type: String
-      config :parse, type: String
+      config :parse, type: [String, NilClass]
+      config :link_names, type: [FalseClass, TrueClass, NilClass]
```

Comment:
Oh right it uses `===`.

But for the default I think it's important we don't send the option. Because the defaults are different based on the parse mode (yes Slack api is a bit :hankey: )